runtime.txt

```diff
@@ -0,0 +1 @@
+3.7
```

Comment:
Seems like there is still a newline here.

Reply:
I can't see it on my local. This is weird.

Reply:
![image](https://user-images.githubusercontent.com/34674462/83150290-b9feec00-a118-11ea-9078-b21b49771590.png)

Reply:
I would recommend you use Vim for this surgical operation. Some helpful commands [here](https://superuser.com/a/1457411).